arrows/core/detected_object_set_output_kw18.cxx

```diff
@@ -33,12 +33,17 @@
 #include <vital/vital_foreach.h>
 #include <vital/util/tokenize.h>
 
-#include <atomic>
 #include <memory>
 #include <vector>
 #include <fstream>
 #include <time.h>
 
+#if (__GNUC__ == 4 && __GNUC_MINOR__ < 5 )
+  #include <cstdatomic>
```

Comment:
Clang also defines those flags, it can be fixed by adding a #ifndef __clang__ clause, however, in addition to the __GNUC__ checks

(I still like having it because everything else builds on C6 default gcc and I use it a bit)